File: Programming Basics with JS/01. First Steps in Coding Exersice/smallShop.js
function solve(arg, arg2, arg3) {
    let product = String(arg);
    let city = String(arg2);
    let quantity = Number(arg3);
    let price = 0;

    switch (city) {
        case "Sofia":
            switch (product) {
                case "coffee":
                    price = quantity * 0.5;
                    break;
                case "water":
                    price = quantity * 0.8
                    break;
                case "beer":
                    price = quantity * 1.2
                    break;
                case "sweets":
                    price = quantity * 1.45
                    break;
                case "peanuts":
                    price = quantity * 1.6
                    break;
            }
            console.log(price);
            break;
        case "Plovdiv":
            switch (product) {
                case "coffee":
                    price = quantity * 0.4;
                    break;
                case "water":
                    price = quantity * 0.7
                    break;
                case "beer":
                    price = quantity * 1.15
                    break;
                case "sweets":
                    price = quantity * 1.30
                    break;
                case "peanuts":
                    price = quantity * 1.5
                    break;
            }
            console.log(price);
            break;
        case "Varna":
            switch (product) {
                case "coffee":
                    price = quantity * 0.45;
                    break;
                case "water":
                    price = quantity * 0.7
                    break;
                case "beer":
                    price = quantity * 1.1
                    break;
                case "sweets":
                    price = quantity * 1.35
                    break;
                case "peanuts":
                    price = quantity * 1.55
                    break;
            }
            console.log(price);
    }
}

solve("sweets",
"Sofia",
"2.23")

File: Programming Basics with Java/03. Conditional Statements Advanced - Lab/src/SmallShop.java
import java.util.Scanner;

public class SmallShop {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String product = scan.nextLine();
        String city = scan.nextLine();
        double amount = Double.parseDouble(scan.nextLine());
        double price = 0;

        if(city.equals("Sofia")){
            switch (product){
                case "coffee":
                    price = amount * 0.5;
                    break;
                case "water":
                    price = amount * 0.8;
                    break;
                case "beer":
                    price = amount * 1.2;
                    break;
                case "sweets":
                    price = amount * 1.45;
                    break;
                case "peanuts":
                    price = amount * 1.6;
                    break;
            }
        } else if(city.equals("Plovdiv")){
            switch (product){
                case "coffee":
                    price = amount * 0.4;
                    break;
                case "water":
                    price = amount * 0.7;
                    break;
                case "beer":
                    price = amount * 1.15;
                    break;
                case "sweets":
                    price = amount * 1.3;
                    break;
                case "peanuts":
                    price = amount * 1.5;
                    break;
            }
        } else if(city.equals("Varna")){
            switch (product){
                case "coffee":
                    price = amount * 0.45;
                    break;
                case "water":
                    price = amount * 0.7;
                    break;
                case "beer":
                    price = amount * 1.1;
                    break;
                case "sweets":
                    price = amount * 1.35;
                    break;
                case "peanuts":
                    price = amount * 1.55;
                    break;
            }
        }

        System.out.println(price);
    }
}
